.github/workflows/update-koryphe-version.yaml

```
@@ -0,0 +1,42 @@
+name: Update Korpyhe Version
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Korpyhe Version'
+        required: true
+
+jobs:
+  update-koryphe-version:
+    runs-on: ubuntu-latest
+    env:
+      MAVEN_OPTS: -Dmaven.wagon.http.retryHandler.count=3 -Dmaven.wagon.httpconnectionManager.ttlSeconds=25
+      VERSION_UPDATE_BRANCH: update-koryphe-version-${{ github.event.inputs.version }}
+    steps:
+
+      - name: Checkout develop
+        uses: actions/checkout@v2
+        with:
+          ref: develop
+          token: ${{ secrets.ADMIN_GITHUB_TOKEN }}
+
+      - name: Set up Github credentials
+        run: |
+          git config user.name Gaffer
+          git config user.email github-actions@github.com
+
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+
+      - name: Update Korpyhe Version
+        run: ./cd/updateKorypheVersion.sh ${{ github.event.inputs.version }}
+
+      - name: Submit PR
+        if: ${{ success() }}
+        uses: repo-sync/pull-request@v2
+        with:
+          source_branch: ${{ env.VERSION_UPDATE_BRANCH }}
+          destination_branch: develop
+          github_token: ${{ secrets.ADMIN_GITHUB_TOKEN }}
```

Don't think admin token would be required for this. You could use the default one:
```suggestion
          github_token: ${{ secrets.GITHUB_TOKEN }}
```